ucrm_install_debian.sh

```diff
@@ -38,6 +56,38 @@ if [ ! -f /home/$UCRM_USER/docker-compose.yml ]; then
 	echo "Replacing env in docker compose"
 	sed -i -e "s/POSTGRES_PASSWORD=ucrmdbpass1/POSTGRES_PASSWORD=$POSTGRES_PASSWORD/g" /home/$UCRM_USER/docker-compose.env
 	sed -i -e "s/SECRET=changeThisSecretKey/SECRET=$SECRET/g" /home/$UCRM_USER/docker-compose.env
+
+	while true; do
+		read -r -p "U CRM has 8080 as default web port, do you like to change to 80? [y/N]: " DOCKER_CHANGE_8080_PORT
```

Comment:
What about this?

"Do you want UCRM to be accessible on port 80? [y/N] (Yes: recommended for most users, No: will set 8080 as default): "